scipy/stats/_morestats.py

```diff
@@ -3923,3 +3923,98 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate', *,
     if not normalize:
         res *= (high-low)/(2.*pi)  # [1] (2.3.14) w/ (2.3.7)
     return res
+
+
+def directionalmean(samples, *, axis=0, normalize=True):
```

Comment:
Ideally we should call this `directional_mean` but since other functions don't use a `_` I guess this is fine. Just wanted to leave this note there.